masterlist.yaml

```diff
@@ -8913,6 +8920,8 @@ plugins:
         subs: [ 'https://forums.nexusmods.com/index.php?/topic/6988461-caco-compatibility/?p=63575286' ]
       - <<: *patch3rdParty_LotD
         condition: 'active("LegacyoftheDragonborn.esm") and not active("DBM_CACO_Patch.esp")'
+      - <<: *patch3rdParty_ClockA
+        condition: 'active("Clockwork.esp") and not (active("ClockworkSorting_Ing_CACO.esp") or active("ClockworkSorting_Ing_CCCurios_CACO.esp") or active("ClockworkSorting_Food_CACO.esp"))'
```

Comment:
You can view the errors that are causing an error here.

![image](https://user-images.githubusercontent.com/1944639/127561291-bffec3a8-554e-46ed-a022-25c60eeed6f0.png)

![image](https://user-images.githubusercontent.com/1944639/127561377-5de2901d-d37c-4e06-893d-4ebea55dc027.png)


ERROR: yaml-cpp: 
error at line 8923, column 9: bad conversion: invalid condition syntax: 
Failed to parse condition 
"active("Clockwork.esp") and not (active("ClockworkSorting_Ing_CACO.esp") or active("ClockworkSorting_Ing_CCCurios_CACO.esp") or active("ClockworkSorting_Food_CACO.esp"))".

Details:
The parser did not consume the following input:
" and not (active("ClockworkSorting_Ing_CACO.esp") or active("ClockworkSorting_Ing_CCCurios_CACO.esp") or active("ClockworkSorting_Food_CACO.esp"))"

Reply:
Yeah, this is, without a doubt, my first time working with automated checkers. Thanks for letting me know!

Reply:
It's showing errors when I've tested locally.

![image](https://user-images.githubusercontent.com/1944639/127623326-18d11426-92ad-456f-b502-ad81b987b153.png)

You can test your updates by pointing LOOT to your repository and branch.
![image](https://user-images.githubusercontent.com/1944639/127624452-67b64b23-cdb7-49a6-9b7c-e0304f477238.png)

Reply:
Oh and you don't really need to squash and merge, I can do that when I merge the pull request.

Reply:
I was on mobile when I applied the fix the second time. Looks like I accidentally removed a parenthesis. (I _did_ technically say that before then.)

Once more I can confirm that this one (commit [`9412337`](https://github.com/BellCubeDev/skyrimse/commit/9412337b1fa3936e6d6f823a52f04c32bc545fd2)) also plays nicely.